src/main/java/net/troja/eve/esi/auth/OAuth.java

```diff
@@ -26,25 +26,30 @@
     private static final String URI_AUTHENTICATION = URI_OAUTH + "/authorize";
     private static final String URI_ACCESS_TOKEN = URI_OAUTH + "/token";
 
-    private String accessToken;
-    private long validUntil;
     private String refreshToken;
     private String clientId;
     private String clientSecret;
     private OAuth2CodeGrantFlow oAuthFlow;
-    private final Map<String, AccessTokenData> accessTokenCache = new HashMap<>();
+    private static final Object STATIC_LOCK = new Object();
```

Comment:
Purely convention, but an alternative pattern is to instead lock on the class, e.g.
synchronized (OAuth.class) { ... do some stuff ... }